### DISCUSSION TITLE
How to query the list of embedded object from collection

### DISCUSSION THREAD

**Discussion by BalasubramaniSundaram95** [isar/isar#1551] [Help]
Hi Isar Git Fam, 

I'm using the 4.0.0-dev.14 version in dev and am struck in querying the List of embedded object from collection.

```
@Collection
class FirstStandard{
    final List<Student> students;
}

@embedded
class Student{
  .....
  .....
  .....
  List<Subject> subjects; 
}
```

Tried the like below

`isar.firstStandard.students --> It's returns the list of students but it's doesn't contain the query or filter property `

How to query the student from collection ?

**Comment by Bohne13** [comment]
This could maybe close to your problem. #1547 

you can start a query like:
```dart
isar.firstStandards.where().studentProperty().findAll() // returns a List<Student>
```

**Reply by BalasubramaniSundaram95** [reply]
Hi @Bohne13 , 

How to watch the particular student object ?

**Reply by Bohne13** [reply]
I haven't used the possibility to watch an object.
But I can strongly recommend reading the documentation.
It's only a breef introduction, but from there you might figure it out yourself.
